### ISSUE TITLE
The "default directory" display breaks if path contains unicode characters

### ISSUE THREAD

**Issue by LittleVulpix** [qTox/qTox#3236]
qTox 1.3.0 @ Windows but I'm sure it exists on previous versions as well.

![Broken path](http://i.imgur.com/cRiBzdB.png)

**Comment by a68366** [comment]
qTox 1.4.0-22-g899a2eb @ Ubuntu 14.04.4

Seems to be layout problem, not Unicode. It breaks when path is too long.
![1](https://cloud.githubusercontent.com/assets/12549653/14943543/2aca36f4-0fed-11e6-8a9d-3002505fb3fb.png)
![2](https://cloud.githubusercontent.com/assets/12549653/14943544/2ace4910-0fed-11e6-8363-6146ca3a45df.png)

**Comment by LittleVulpix** [comment]
I disagree, namely because the "break" in my case is because the folder is definitely not called ĂƒĂƒĂƒĂƒĂƒĂƒĂƒĂƒĂƒĂƒĂƒĂƒĂƒĂƒĂƒĂƒĂƒ .
